src/lib/connector.ts

```diff
@@ -1,64 +1,44 @@
-import { Signer, utils, ethers } from "ethers";
-import {
-  ConnectionOptions,
-  Connection,
-  Token,
-  SupportedNetwork,
-} from "../interfaces.js";
+import { Signer, ethers } from "ethers";
+import { ConnectionOptions, Connection, Token } from "../interfaces.js";
 import { list } from "./list.js";
 import { createToken } from "./token.js";
-import { query } from "./query.js";
+import { read, write } from "./query.js";
 import { create } from "./create.js";
 import { hash } from "./hash.js";
+import { receipt } from "./tableland-calls.js";
+import { SUPPORTED_NETWORKS, contractAddresses } from "./util.js";
 
 declare let globalThis: any;
 
-const SUPPORTED_NETWORKS: SupportedNetwork[] = [
-  {
-    key: "rinkeby",
-    phrase: "Ethereum Rinkeby",
-  },
-];
-
 export async function getSigner(): Promise<Signer> {
   await globalThis.ethereum.request({ method: "eth_requestAccounts" });
   const provider = new ethers.providers.Web3Provider(globalThis.ethereum);
   const signer = provider.getSigner();
   return signer;
 }
 
-export async function userCreatesToken(signer: Signer): Promise<Token> {
-  const sign = {
-    signMessage: async (message: Uint8Array): Promise<Uint8Array> => {
-      const sig = await signer.signMessage(message);
-      return utils.arrayify(sig);
-    },
-  };
-  const iat = ~~(Date.now() / 1000);
-  const exp = iat + 60 * 60 * 10; // Default to ~10 hours
-
-  // WARN: This is a non-standard JWT
-  // Borrows ideas from: https://github.com/ethereum/EIPs/issues/1341
-  const iss = await signer.getAddress();
-  const network = await signer.provider?.getNetwork();
-  const chain = network?.chainId ?? "unknown";
-  let net = network?.name;
-  if (net?.startsWith("matic")) net = "poly";
-  else net = "eth";
-  const kid = `${net}:${chain}:${iss}`;
+export async function userCreatesToken(
```

Comment:
Formatting fix, and also this is now accessible from outside the lib. It is useful for creating standalone tokens.